hack/generate-embedded-files.sh

```diff
@@ -34,30 +31,21 @@ elif ! [ -x "$(command -v ${LICENSES})" ]; then
     # Also can't be easily installed from a vendor folder because it relies on non-go files
     # from a dependency.
     echo "Installing go-licenses"
-    pushd $(mktemp -d ${TMPDIR:-/tmp}/generate-statik.XXXXXX)
-    #TODO(marlongamez): unpin this version once we're able to build using go 1.16.x
     go mod init tmp; GOBIN=${BIN} go get github.com/google/go-licenses@9376cf9847a05cae04f4589fe4898b9bce37e684
```

Comment:
I think this is failing in the integration test. you will have to move to a temporary directory and run go mod init there.

Reply:
ohhhh.. thank you!! let me try that